core/src/io/anuke/mindustry/world/blocks/distribution/Conveyor.java

```diff
@@ -406,7 +406,7 @@ private ItemPos(){
 
         static long packItem(Item item, float x, float y){
             short[] shorts = packShorts;
-            shorts[0] = (short)item.id;
```

Comment:
While this is correct, most of the code here is going to be completely removed in the `conveyor-optimizations` branch.
I would prefer it if you didn't edit the `Conveyor` class to prevent merge conflicts in the future.

Reply:
Okay 👍 I'll get back it.

Reply:
@Anuken I reverted for that. plz check it out.